### PR TITLE
feat: Add statement descriptors to transaction APIs

### PR DESCRIPTION
**Pull request by cbetta**
Autogenerated code created using the OpenAPI SDK generator.

### CODE REVIEW

File: api/api/openapi.yaml [hunk 1]
```diff
@@ -4770,12 +4850,13 @@ components:
           - categories
           - categories
           sku: sku-789123
-        currency: USD
         is_subsequent_payment: true
         store: true
+        intent: capture
+        statement_descriptor: ""
```
Comment (bloowitt) [inline api/api/openapi.yaml:4856]: I'm not sure but should this not be either null or an object with stuff inside?

Comment (cbetta) [reply api/api/openapi.yaml:4856]: I am not sure what the generator uses this yam file for tbh. I don't think it's used. When I look at the code generated it seems to reference a model.

I'd be happy to ignore this.

Comment (cbetta) [reply api/api/openapi.yaml:4856]: FYI https://github.com/gr4vy/gr4vy-go/pull/13/files#diff-9366182748ce7ff8768cdc1cfc45deb20181f63a15a8b8b90b5116634e7fa6deR40

Comment (danielknell) [reply api/api/openapi.yaml:4856]: thats also from an `Example` object, those are intended for documentation tools rather than sdk generators.

**Review by bloowitt — APPROVED** [review]

**Pull request merged**